microservices/strapi-connector-ms/config/envConfig.js

```diff
@@ -0,0 +1,65 @@
+const envSchema = {
+    type: 'object',
+    required: ['SERVER_SERVLET_CONTEXT_PATH',
+        'SPRING_DATASOURCE_USERNAME',
+        'SPRING_DATASOURCE_PASSWORD',
+        'SPRING_DATASOURCE_URL',
+        'APP_ORIGIN',
+        'KEYCLOAK_REALM',
+        'KEYCLOAK_AUTH_URL',
+        'KEYCLOAK_CLIENT_SECRET',
+        'KEYCLOAK_CLIENT_ID',
+        'JWT_PUB_KEY'
+    ],
+    properties: {
+        SERVER_SERVLET_CONTEXT_PATH: {
+            type: 'string',
+            default: '/'
+        },
+        SPRING_DATASOURCE_USERNAME: {
+            type: 'string',
+            default: 'entando-strapi-connector-user'
+        },
+        SPRING_DATASOURCE_PASSWORD: {
+            type: 'string',
+            default: 'entando-strapi-connector-password'
+        },
+        SPRING_DATASOURCE_URL: {
+            type: 'string',
+            default: 'jdbc:postgresql://localhost:5432/entando-strapi-connector'
+        },
+        STRAPI_CONFIG_TABLE: {
+            type: 'string',
+            default: 'strapi_url'
+        },
+        APP_ORIGIN: {
+            type: 'string',
+            default: 'http://localhost:3000'
+        },
+        KEYCLOAK_REALM: {
+            type: 'string',
+            default: 'entando-dev'
+        },
+        KEYCLOAK_AUTH_URL: {
+            type: 'string',
+            default: 'http://localhost:9080/auth'
+        },
+        KEYCLOAK_CLIENT_SECRET: {
+            type: 'string',
+            default: 'CFXd3zdoDgnSvLfovOvjFzpFkfnWfFar'
+        },
+        KEYCLOAK_CLIENT_ID: {
+            type: 'string',
+            default: 'entando-strapi-connector-ms-client'
+        },
+        JWT_PUB_KEY :{
```

Comment:
I knew about the well-know configuration but the idea was to give the user a way to validate the tokens without the MS having to execute any call to keycloak.

The value must be configured by who install the service IF they don't want to use the introspection endpoint for the verification flow.

In a future release I'd like to implement an option to autoconfigure this value using the well-know endpoint and to use this configuration as a fallback in case of errors with the introspection flow.

For this first relase I wasn't able to implement that due to time constraints, I'm adding this as a note to the PR.